accounts-db/src/tiered_storage/writer.rs

```diff
@@ -38,17 +52,18 @@ impl<'format> TieredStorageWriter<'format> {
         V: Borrow<Hash>,
     >(
         &self,
-        _accounts: &StorableAccountsWithHashesAndWriteVersions<'a, 'b, T, U, V>,
-        _skip: usize,
+        accounts: &StorableAccountsWithHashesAndWriteVersions<'a, 'b, T, U, V>,
+        skip: usize,
     ) -> TieredStorageResult<Vec<StoredAccountInfo>> {
-        let footer = TieredStorageFooter {
+        let mut footer = TieredStorageFooter {
             account_meta_format: self.format.account_meta_format,
             owners_block_format: self.format.owners_block_format,
             account_block_format: self.format.account_block_format,
             account_index_format: self.format.account_index_format,
             ..TieredStorageFooter::default()
         };
 
+        footer.account_entry_count = accounts.accounts.len().saturating_sub(skip) as u32;
```

Comment:
Sure thing.  Will fix it in this PR.  But in reality, we want to do it based on the actual accounts being written so it will still need `mut` in the future (for instance, if the input vector is too large for tiered-storage's index block or something).

Reply:
Done!